lib/game_runner.rb

```diff
@@ -0,0 +1,146 @@
+        end
+    end
+
+    def game_options
+        user_is_active = true
+        while user_is_active
+            input = home_menu
+            puts `clear`
+            case input
+            when '1'
+                set_up
+                play_crazy_eights(false)
+            when '2'
+                set_up
+                @game.deal_start_hand('computer')
+                play_crazy_eights(true)
+            when '3' then rules
+            when '4'
+                puts 'Goodbye!'
+                user_is_active = false
+            else
+                puts "Mmm, I'm getting mixed signals. Can you try making a "
+                puts 'selection again? Just enter 1, 2, or 3 please.'
+            end
+        end
+    end
+
+    def home_menu
+        puts
+        puts "To start a single player game, [enter] '1'"
+        puts "To play against the computer, [enter] '2'"
+        puts "To view the rules, [enter] '3'"
+        puts "To exit, [enter] '4'"
+        puts
+        input = STDIN.gets.chomp.strip
+        input
+    end
+
+    def set_up
+        @game = CrazyEightGame.create(player_id: @player.id, turn_count: 0)
+        @game.new_deck
+        @game.deal_start_hand(@player.id)
+        @game.place_start_card
+    end
+
+    def game_over?(computer)
+        @game.nil? || @game.remaining.zero? || @game.player_hand(@player.id).length.zero? || computer && @game.player_hand('computer').length.zero? 
+    end
+
+    def play_crazy_eights(computer)
+        until game_over?(computer)
+            if computer && @game.turn_count % 2 == 1 
+                @game.computer_turn
+            else
+                @game.view_top_card
+                @game.view_hand(@player.id)
+                choice = turn_options.upcase
+                puts `clear`
+                case choice
+                when '1'then @game.draw_card(@player.id)
+                when '2'then rules
+                when '3'
+                    exit_game_and_delete_deck(@game)
+                else @game.play_card(@player, choice)
+                end
+            end
+        end
+        check_for_winner(computer)
+    end
+
+    def turn_options
+        puts
+        puts 'What would you like to do?'
+        puts
+        puts 'To play a card, [enter] the card code:'
+        puts "To draw a card, [enter] '1'"
+        puts "To review the rules, [enter] '2'"
+        puts "To exit and end this game, [enter] '3'"
+        puts
+        choice = STDIN.gets.strip
+        choice
+    end
+
+    def exit_game_and_delete_deck(game)
+        puts 'Are you sure you want to exit and end this game?'
+        puts "[enter] 'yes' to confirm."
+        input = STDIN.gets.strip.downcase
+        case input
+        when 'yes'
+            Hand.where(deck_id: game.deck_id).destroy_all
+            game.destroy
+            @game = nil
+        end
+    end
+
+    def winning_turns(computer)
+        computer ? @game.turn_count/2 + 1 : @game.turn_count
```

Comment:
```suggestion
        computer ? @game.turn_count / 2 + 1 : @game.turn_count
```
Spaces should be used between operators and values
https://rubystyle.guide/#spaces-operators